tests/signature/test_parsing.py

```diff
@@ -14,9 +16,22 @@
 from litestar.types.helper_types import OptionalSequence
 from litestar.utils.signature import ParsedSignature
 
-if TYPE_CHECKING:
-    from types import ModuleType
-    from typing import Callable
+
+def create_signature_model_for_test(
```

Comment:
Y? 

Assertions should not be wrapped inside helpers.  

Please remove this and add the assertion to the tests directly.

Reply:
Because it was the path of least resistance to getting the check in place for every test. I don't think I'll have the bandwidth to change this with the other stuff I have to do. I'll close the PR.

Reply:
The PR was really only opened to demonstrate the issue anyway.